libs/go.json

```diff
@@ -94,10 +94,13 @@
   {"id":"go.telegram-bot-api","imports":["github.com/Syfaro/telegram-bot-api"],"name":"telegram-bot-api","repo":"Syfaro/telegram-bot-api","tags":["telegram"],"tech":["chat-ux"],"status":"model-ready"},
   {"id":"go.cli","imports":["github.com/urfave/cli-apps"],"name":"cli","repo":"urfave/cli","tags":["cli-apps"],"tech":["cli"],"status":"awaiting-model"},
   {"id":"go.cobra","imports":["github.com/spf13/cobra"],"name":"Cobra","repo":"spf13/cobra","tags":["cli-apps"],"tech":["cli"],"status":"awaiting-model"},
+  {"id":"go.complete","imports":["github.com/posener/complete"],"name":"complete","repo":"posener/complete","tags":["bash-completion"],"tech":["cli"],"status":"awaiting-model"},
```

Comment:
is that a library? seems like a bash tool

Reply:
seems so, they have an example https://github.com/posener/complete/blob/v1/example/self/main.go